wp-includes/version.php

```diff
@@ -4,7 +4,7 @@
  *
  * @global string $wp_version
  */
-$wp_version = '4.8.2';
+$wp_version = '4.8.3';
```

Comment:
@greg-1-anderson this looks like an update to `4.8.3` not `4.9`

Reply:
Drat.

Reply:
I've got 4.9 on the Pantheon update site...

Reply:
```
terminus env:commit wp-update.update --yes '--message=Update to WordPress 4.9. For more information, see '
 [warning] There is no code to commit.
```
☹️